src/main/resources/application.yml

```diff
@@ -19,7 +19,7 @@ ams:
     enabled: true
 
 paygops:
-  base-url: https://feature-test1.paygops.com/
```

Comment:
Why change this? Wasn't this already following the convention as per the design doc that was made?